libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.scss

```diff
@@ -3,3 +3,7 @@
 .sky-data-manager-column-picker-content sky-repeater-item-content {
   margin-top: $sky-margin-inline-compact;
 }
+
+.sky-data-manager-column-picker-footer-stacked-indicator {
+  margin: 0px 5px 10px 0px;
```

Comment:
You should use the appropriate SCSS variables for the `5px` and `10px` values (see the class right above it for reference). Also you don't need to specify `px` when the value is `0` since `0` is the same in any unit (e.g. `margin: 0 $some-var $some-other-var 0;`

Reply:
fixed in commit: [use scss variables](https://github.com/blackbaud/skyux/pull/349/commits/4e57053cf6e5bed05e7d0196b4fe42f2cddf002c)